src/Build/Definition/Project.cs

```diff
@@ -2763,7 +2763,7 @@ private static IEnumerable<ProjectItemElement> GetItemElementsThatMightAffectIte
                         itemElement.RemoveLocation == null);
 
                 // add the include operation that created the project item element
-                return [item.Xml, ..relevantElementsAfterInclude];
+                return [item.Xml, .. relevantElementsAfterInclude];
```

Comment:
Is this really the canonical style? I definitely expected the way it was before.

Reply:
It surprised me too, but yes, this is what the rule requested. I recall that I've done this one manually at least on one occasion. Took me a while to figure out what it wanted.
Code modified with this PR passes through build.cmd without warnings so it should be correct - as least as far as the .editorconfig is concerned.